app/gui2/src/stores/suggestionDatabase/index.ts

```diff
@@ -25,6 +25,7 @@ export class SuggestionDb extends ReactiveDb<SuggestionId, SuggestionEntry> {
     }
     return []
   })
+  conflictingNames = new ReactiveIndex(this, (id, entry) => [[entry.name, id]])
```

Comment:
Not sure what you mean here, but `nameToId` index is not suitable here, because it stores FQNs. We don’t have any other index that operates on entry names.

Reply:
I meant: we could make index named `nameIndex` instead of a bit confusing `conflictingNames`.